### ISSUE TITLE
uTorrent - Post Processing stopping after Auto Section: Couchpotato

### ISSUE THREAD

**Issue by blade316** [clinton-hall/nzbToMedia#1066]
Hey guys,

Hopefully you can help me, I've just set up nzbToMedia, NZB's using SABnzbd and Couchpotato are fine, however I cant get uTorrent to successfully post process, heres my set up:

Windows 8.1 64bit
Plex, Couchpotato and uTorrent are all on the same machine
I have 8 HDD's in JBOD using storage spaces running ReFS
The Storage Volume is D: drive

Post Processing Folder = "C:\PostProcessingScripts\nzbToMedia\"

**Plex Config**
Movies Folder = "D:\Movies"
TV Folder = "D:\TV"
Music Folder = "D:\Music"

**CouchPotato Config**
Version = 3.0.1
ID = 3232
NZB Downloader = SABnzbd
NZB Label = movies
Torrent Downloader = uTorrent
Torrent label = movies
Remove the torrent from uTorrent after it finished seeding = enabled
Also remove the leftover files = enabled
Delete a release after the download has failed = enabled
Folder where CP searches for movies = "D:\Downloads\Completed\Movies"
Default folder where the movies are moved/copied/linked to = "D:\Movies"

**uTorrent Config**
version = 3.4.2 Build 37754
WebUI = enabled
Append torrent label = enabled
Put new downloads in = "D:\Downloads\Clients\uTorrent\Incomplete"
Move completed downloads to = "D:\Downloads\Completed"
Automatically load torrents from = "D:\Downloads\Clients\uTorrent\blackhole" (not that i use it)
Run this program when a torrent changes state
C:\PostProcessingScripts\nzbToMedia\TorrentToMedia.py "%D" "%N" "%L" "%I"

**Problem:**
Once the torrent finishes downloading and changes state, the TorrentToMedia.py runs, and looks like its pulling all the correct arguments through, however it never gets passed:

::MAIN: Auto-detected SECTION:CouchPotato

**autoProcessMedia.cfg**

```
[General]
    # Enable/Disable update notifications
    version_notify = 1
    # Enable/Disable automatic updates
    auto_update = 1
    # Set to the full path to the git executable
    git_path = C:\Program Files\Git\git-cmd.exe
    # GitHUB user for repo
    git_user = ""
    # GitHUB branch for repo
    git_branch = nightly
    # Enable/Disable forceful cleaning of leftover files following postprocess 
    force_clean = 1
    # Enable/Disable logging debug messages to nzbtomedia.log
    log_debug = 1
    # Enable/Disable logging database messages to nzbtomedia.log
    log_db = 1
    # Enable/Disable logging environment variables to debug nzbtomedia.log (helpful to track down errors calling external tools.)
    log_env = 1
    # Enable/Disable logging git output to debug nzbtomedia.log (helpful to track down update failures.)
    log_git = 1
    # Set to the directory where your ffmpeg/ffprobe executables are located
    ffmpeg_path = C:\ffmpeg\bin
    # Enable/Disable media file checking using ffprobe.
    check_media = 1
    # Enable/Disable a safety check to ensure we don't process all downloads in the default_downloadDirectories by mistake.
    safe_mode = 1

[Posix]
    ### Process priority setting for External commands (Extractor and Transcoder) on Posix (Unix/Linux/OSX) systems.
    # Set the Niceness value for the nice command. These range from -20 (most favorable to the process) to 19 (least favorable to the process).
    niceness = 0
    # Set the ionice scheduling class. 0 for none, 1 for real time, 2 for best-effort, 3 for idle.
    ionice_class = 0
    # Set the ionice scheduling class data. This defines the class data, if the class accepts an argument. For real time and best-effort, 0-7 is valid data.
    ionice_classdata = 0

[CouchPotato]
    #### autoProcessing for Movies
    #### movie - category that gets called for post-processing with CPS
    [[movie]]
        enabled = 0
        apikey = ""
        host = localhost
        port = 5050
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        ssl = 0
        web_root = ""
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 0
        keep_archive = 1
        method = renamer
        delete_failed = 0
        wait_for = 2
        extract = 1
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 0
        ##### Enable if Couchpotato is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""
    [[movies]]
        enabled = 1
        apikey = 9d57ffc3935241049d62f41930517681
        host = localhost
        port = 8082
        ssl = 0
        web_root = ""
        Torrent_NoLink = 0
        keep_archive = 1
        method = renamer
        delete_failed = 1
        wait_for = 2
        extract = 0
        minSize = 100
        delete_ignored = 1
        remote_path = 0
        watch_dir = D:\Downloads\Completed\Movies

[SickBeard]
    #### autoProcessing for TV Series
    #### tv - category that gets called for post-processing with SB
    [[tv]]
        enabled = 0
        host = localhost
        port = 8081
        username = ""
        password = ""
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        web_root = ""
        ssl = 0
        fork = auto
        delete_failed = 0
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 0
        keep_archive = 1
        process_method = ""
        # force processing of already processed content when running a manual scan.
        force = 0
        # tell SickRage to delete all source files after processing.
        delete_on = 0
        extract = 1
        nzbExtractionBy = Downloader
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 0
        ##### Enable if SickBeard is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""

[NzbDrone]
    #### autoProcessing for TV Series
    #### ndCategory - category that gets called for post-processing with NzbDrone
    [[tv]]
        enabled = 0
        apikey = ""
        host = localhost
        port = 8989
        username = ""
        password = ""
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        web_root = ""
        ssl = 0
        delete_failed = 0
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 0
        keep_archive = 1
        extract = 1
        nzbExtractionBy = Downloader
        wait_for = 2
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 0
        ##### Enable if NzbDrone is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""

[HeadPhones]
    #### autoProcessing for Music
    #### music - category that gets called for post-processing with HP
    [[music]]
        enabled = 1
        apikey = 72d068554a55c18e38549dadd2a0d667
        host = localhost
        port = 8181
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        ssl = 0
        web_root = ""
        wait_for = 2
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 1
        keep_archive = 1
        extract = ""
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 1
        ##### Enable if HeadPhones is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = D:\Downloads\Completed\Music

[Mylar]
    #### autoProcessing for Comics
    #### comics - category that gets called for post-processing with Mylar
    [[comics]]
        enabled = 0
        host = localhost
        port = 8090
        username = ""
        password = ""
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        web_root = ""
        ssl = 0
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 0
        keep_archive = 1
        extract = 1
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 0
        ##### Enable if Mylar is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""

[Gamez]
    #### autoProcessing for Games
    #### games - category that gets called for post-processing with Gamez
    [[games]]
        enabled = 0
        apikey = ""
        host = localhost
        port = 8085
        ###### 
        library = Set to path where you want the processed games to be moved to.
        ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
        ssl = 0
        web_root = ""
        # Enable/Disable linking for Torrents
        Torrent_NoLink = 0
        keep_archive = 1
        extract = 1
        # Set this to minimum required size to consider a media file valid (in MB)
        minSize = 0
        # Enable/Disable deleting ignored files (samples and invalid media files)
        delete_ignored = 0
        ##### Enable if Gamez is on a remote server for this category
        remote_path = 0
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""

[Network]
    # Enter Mount points as LocalPath,RemotePath and separate each pair with '|'
    # e.g. MountPoints = /volume1/Public/,E:\|/volume2/share/,\\NAS\
    mount_points = ""

[Nzb]
    ###### clientAgent - Supported clients: sabnzbd, nzbget
    clientAgent = sabnzbd
    ###### SabNZBD (You must edit this if your using nzbToMedia.py with SabNZBD)
    sabnzbd_host = http://mediapc
    sabnzbd_port = 8090
    sabnzbd_apikey = 5e0a9666440541a2a4585c84562cbdfd
    ###### Enter the default path to your default download directory (non-category downloads). this directory is protected by safe_mode.
    default_downloadDirectory = D:\Downloads\Clients\SABnzbd\Complete

[Torrent]
    ###### clientAgent - Supported clients: utorrent, transmission, deluge, rtorrent, vuze, other
    clientAgent = utorrent
    ###### useLink - Set to hard for physical links, sym for symbolic links, move to move, move-sym to move and link back, and no to not use links (copy)
    useLink = move-sym
    ###### outputDirectory - Default output directory (categories will be appended as sub directory to outputDirectory)
    outputDirectory = D:\Downloads\Completed
    ###### Enter the default path to your default download directory (non-category downloads). this directory is protected by safe_mode. 
    default_downloadDirectory = D:\Downloads\Clients\uTorrent\Complete
    ###### Other categories/labels defined for your downloader. Does not include CouchPotato, SickBeard, HeadPhones, Mylar categories.
    categories = pr0n, software
    ###### A list of categories that you don't want to be flattened (i.e preserve the directory structure when copying/linking.
    noFlatten = pictures, manual
    ###### uTorrent Hardlink solution (You must edit this if your using TorrentToMedia.py with uTorrent)
    uTorrentWEBui = http://localhost:8000/gui/
    uTorrentUSR = admin
    uTorrentPWD = 1234
    ###### Transmission (You must edit this if your using TorrentToMedia.py with Transmission)
    TransmissionHost = localhost
    TransmissionPort = 9091
    TransmissionUSR = your username
    TransmissionPWD = your password
    #### Deluge (You must edit this if your using TorrentToMedia.py with deluge. Note that the host/port is for the deluge daemon, not the webui)
    DelugeHost = localhost
    DelugePort = 58846
    DelugeUSR = your username
    DelugePWD = your password
    ###### ADVANCED USE - ONLY EDIT IF YOU KNOW WHAT YOU'RE DOING ######
    deleteOriginal = 0
    chmodDirectory = 0
    resume = 1
    resumeOnFailure = 1

[Extensions]
    compressedExtensions = .zip, .rar, .7z, .gz, .bz, .tar, .arj, .1, .01, .001
    mediaExtensions = .mkv, .avi, .mov, .mp4, .m4v
    audioExtensions = .mp3, .aac, .ogg, .ape, .m4a, .asf, .wma, .flac
    metaExtensions = .nfo, .sub, .srt, .jpg, .gif, .sub, .idx

[Plex]
    # Only enter these details if you want to update plex library after processing.
    # Do not enter these details if you send the plex notifications from Sickbeard/CouchPotato.
    plex_host = localhost
    plex_port = 32400
    plex_token = ""
    plex_ssl = 0
    # Enter Plex category to section mapping as Category,section and separate each pair with '|'
    # e.g. plex_sections = movie,3|tv,4
    plex_sections = ""

[Transcoder]
    # getsubs. enable to download subtitles.
    getSubs = 0
    # subLanguages. create a list of languages in the order you want them in your subtitles. 
    subLanguages = eng, spa, fra
    # transcode. enable to use transcoder
    transcode = 0
    ###### duplicate =1 will create a new file. =0 will replace the original
    duplicate = 1
    # concat. joins cd1 cd2 etc into a single video.
    concat = 1
    # IgnoreExtensions is a comma-separated list of extensions that will not be transcoded.
    ignoreExtensions = .avi, .mkv, .mp4
    # outputFastStart. 1 will use -movflags + faststart. 0 will disable this from being used.
    outputFastStart = 0
    # outputQualityPercent. used as -q:a value. 0 will disable this from being used.
    outputQualityPercent = 0
    # outputVideoPath. Set path you want transcoded videos moved to. Leave blank to disable.
    outputVideoPath = ""
    # processOutput. 1 will send the outputVideoPath to SickBeard/CouchPotato. 0 will send original files.
    processOutput = 0
    # audioLanguage. set the 3 letter language code you want as your primary audio track.
    audioLanguage = eng
    # allAudioLanguages. 1 will keep all audio tracks (uses AudioCodec3) where available.
    allAudioLanguages = 0
    # allSubLanguages. 1 will keep all existing sub languages. 0 will discard those not in your list above.
    allSubLanguages = 0
    # embedSubs. 1 will embed external sub/srt subs into your video if this is supported.
    embedSubs = 1
    # burnInSubtitle. burns the default sub language into your video (needed for players that don't support subs)
    burnInSubtitle = 0
    # extractSubs. 1 will extract subs from the video file and save these as external srt files.
    extractSubs = 0
    # externalSubDir. set the directory where subs should be saved (if not the same directory as the video)
    externalSubDir = ""
    # hwAccel. 1 will set ffmpeg to enable hardware acceleration (this requires a recent ffmpeg)
    hwAccel = 0
    # generalOptions. Enter your additional ffmpeg options here with commas to separate each option/value (i.e replace spaces with commas).
    generalOptions = ""
    # outputDefault. Loads default configs for the selected device. The remaining options below are ignored.
    # If you want to use your own profile, leave this blank and set the remaining options below.
    # outputDefault profiles allowed: iPad, iPad-1080p, iPad-720p, Apple-TV2, iPod, iPhone, PS3, xbox, Roku-1080p, Roku-720p, Roku-480p, mkv, mp4-scene-release
    outputDefault = ""
    #### Define custom settings below.
    outputVideoExtension = .mp4
    outputVideoCodec = libx264
    VideoCodecAllow = ""
    outputVideoPreset = medium
    outputVideoResolution = 1920:1080
    outputVideoFramerate = 24
    outputVideoBitrate = 800000
    outputVideoCRF = 19
    outputVideoLevel = 3.1
    outputAudioCodec = ac3
    AudioCodecAllow = ""
    outputAudioChannels = 6
    outputAudioBitrate = 640k
    outputAudioTrack2Codec = libfaac
    AudioCodec2Allow = ""
    outputAudioTrack2Channels = 2
    outputAudioTrack2Bitrate = 128000
    outputAudioOtherCodec = libmp3lame
    AudioOtherCodecAllow = ""
    outputAudioOtherChannels = ""
    outputAudioOtherBitrate = 128000
    outputSubtitleCodec = ""

[WakeOnLan]
    ###### set wake = 1 to send WOL broadcast to the mac and test the server (e.g. xbmc) the host and port specified.
    wake = 0
    host = 192.168.1.37
    port = 80
    mac = 00:01:2e:2D:64:e1

[UserScript]
    #Use user_script for uncategorized downloads
    #Set the categories to use external script.
    #Use "UNCAT" to process non-category downloads, and "ALL" for all defined categories.
    [[UNCAT]]
        #Enable/Disable this subsection category
        enabled = 0
        Torrent_NoLink = 0
        keep_archive = 1
        extract = 1
        #Enable if you are sending commands to a remote server for this category
        remote_path = 0
        #What extension do you want to process? Specify all the extension, or use "ALL" to process all files.
        user_script_mediaExtensions = .mkv, .avi, .divx, .xvid, .mov, .wmv, .mp4, .mpg, .mpeg
        #Specify the path to your custom script. Use "None" if you wish to link this category, but NOT run any external script.
        user_script_path = /nzbToMedia/userscripts/script.sh
        #Specify the argument(s) passed to script, comma separated in order.
        #for example FP,FN,DN, TN, TL for file path (absolute file name with path), file name, absolute directory name (with path), Torrent Name, Torrent Label/Category.
        #So the result is /media/test/script/script.sh FP FN DN TN TL. Add other arguments as needed eg -f, -r
        user_script_param = FN
        #Set user_script_runOnce = 0 to run for each file, or 1 to only run once (presumably on the entire directory).
        user_script_runOnce = 0
        #Specify the successcodes returned by the user script as a comma separated list. Linux default is 0
        user_script_successCodes = 0
        #Clean after? Note that delay function is used to prevent possible mistake :) Delay is intended as seconds
        user_script_clean = 1
        delay = 120
        #Unique path (directory) created for every download. set 0 to disable.
        unique_path = 1
        ##### Set to path where download client places completed downloads locally for this category
        watch_dir = ""

[ASCII]
    #Set convert =1 if you want to convert any "foreign" characters to ASCII (UTF8) before passing to SB/CP etc. Default is disabled (0).
    convert = 1

[passwords]
    # enter the full path to a text file containing passwords to be used for extraction attempts.
    # In the passwords file, every password should be on a new line
    PassWordFile = ""

[Custom]
    # enter a list (comma separated) of Group Tags you want removed from filenames to help with subtitle matching.
    # e.g remove_group = [rarbag],-NZBgeek
    # be careful if your "group" is a common "real" word. Please report if you have any group replacements that would fall in this category.
    remove_group = ""
```

Post Processing Log below:

```
2016-06-18 15:37:26 INFO    ::MAIN: Loading config from [C:\PostProcessingScripts\nzbToMedia\autoProcessMedia.cfg]
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: TMP: C:\Users\xbmc\AppData\Local\Temp
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: COMPUTERNAME: MEDIAPC
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: USERDOMAIN: MEDIAPC
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PSMODULEPATH: C:\Windows\system32\WindowsPowerShell\v1.0\Modules\
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: COMMONPROGRAMFILES: C:\Program Files (x86)\Common Files
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROCESSOR_IDENTIFIER: Intel64 Family 6 Model 42 Stepping 7, GenuineIntel
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROGRAMFILES: C:\Program Files (x86)
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROCESSOR_REVISION: 2a07
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PATH: C:\Program Files\Adaptec\maxView Storage Manager\ssl\bin;C:\ProgramData\Oracle\Java\javapath;C:\Program Files\Adaptec\maxView Storage Manager\pegasus\bin;C:\Program Files\Adaptec\maxView Storage Manager\;C:\Program Files (x86)\Windows Resource Kits\Tools\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x64;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files (x86)\cURL\bin;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x64;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: SYSTEMROOT: C:\Windows
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: OPENSSL_BIN: C:\Program Files\Adaptec\maxView Storage Manager\\ssl\bin
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: OPENSSL_HOME: C:\Program Files\Adaptec\maxView Storage Manager\ssl
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROGRAMFILES(X86): C:\Program Files (x86)
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: COMSPEC: C:\Windows\system32\cmd.exe
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: MAXVIEW_INSTALLDIR: C:\Program Files\Adaptec\maxView Storage Manager\
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: TEMP: C:\Users\xbmc\AppData\Local\Temp
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: COMMONPROGRAMFILES(X86): C:\Program Files (x86)\Common Files
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROCESSOR_ARCHITECTURE: x86
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: ALLUSERSPROFILE: C:\ProgramData
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PEGASUS_HOME: C:\Program Files\Adaptec\maxView Storage Manager\pegasus
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: LOCALAPPDATA: C:\Users\xbmc\AppData\Local
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: HOMEPATH: \Users\xbmc
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: USERDOMAIN_ROAMINGPROFILE: MEDIAPC
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROGRAMW6432: C:\Program Files
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: USERNAME: xbmc
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: LOGONSERVER: \\MEDIAPC
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: SESSIONNAME: Console
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PROGRAMDATA: C:\ProgramData
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: PATHEXT: .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: ASL.LOG: Destination=file
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: FP_NO_HOST_CHECK: NO
2016-06-18 15:37:26 INFO    ::ENVIRONMENT: WINDIR: C:\Windows
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: APPDATA: C:\Users\xbmc\AppData\Roaming
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: HOMEDRIVE: C:
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: PEGASUS_ROOT: C:\Program Files\Adaptec\maxView Storage Manager\pegasus
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: SYSTEMDRIVE: C:
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: NUMBER_OF_PROCESSORS: 4
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: PROCESSOR_LEVEL: 6
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: PROCESSOR_ARCHITEW6432: AMD64
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: JRE_HOME: C:\Program Files\Adaptec\maxView Storage Manager\\jre
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: COMMONPROGRAMW6432: C:\Program Files\Common Files
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: OS: Windows_NT
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: PUBLIC: C:\Users\Public
2016-06-18 15:37:27 INFO    ::ENVIRONMENT: USERPROFILE: C:\Users\xbmc
2016-06-18 15:37:27 INFO    ::MAIN: Checking database structure...
2016-06-18 15:37:27 DEBUG   ::MAIN: Checking Initial Schema database upgrade
2016-06-18 15:37:27 DB      ::MAIN: nzbtomedia.db: SELECT 1 FROM sqlite_master WHERE name = ?; with args ('db_version',)
2016-06-18 15:37:27 DB      ::MAIN: nzbtomedia.db: SELECT db_version FROM db_version
2016-06-18 15:37:27 DEBUG   ::MAIN: InitialSchema upgrade not required
2016-06-18 15:37:27 DEBUG   ::MAIN: Checking if we can use git commands: "C:\Program Files\Git\git-cmd.exe" version
2016-06-18 15:37:27 DEBUG   ::MAIN: Executing "C:\Program Files\Git\git-cmd.exe" version with your shell in C:\PostProcessingScripts\nzbToMedia
2016-06-18 15:37:27 DEBUG   ::MAIN: git output: 'version' is not recognized as an internal or external command,
operable program or batch file.

C:\PostProcessingScripts\nzbToMedia>
2016-06-18 15:37:27 DEBUG   ::MAIN: "C:\Program Files\Git\git-cmd.exe" version : returned successful
2016-06-18 15:37:27 DEBUG   ::MAIN: Using: "C:\Program Files\Git\git-cmd.exe"
2016-06-18 15:37:27 DEBUG   ::MAIN: Executing "C:\Program Files\Git\git-cmd.exe" symbolic-ref -q HEAD with your shell in C:\PostProcessingScripts\nzbToMedia
2016-06-18 15:37:27 DEBUG   ::MAIN: git output: 'symbolic-ref' is not recognized as an internal or external command,
operable program or batch file.

C:\PostProcessingScripts\nzbToMedia>
2016-06-18 15:37:27 DEBUG   ::MAIN: "C:\Program Files\Git\git-cmd.exe" symbolic-ref -q HEAD : returned successful
2016-06-18 15:37:27 INFO    ::MAIN: Checking if git needs an update
2016-06-18 15:37:27 DEBUG   ::MAIN: Executing "C:\Program Files\Git\git-cmd.exe" rev-parse HEAD with your shell in C:\PostProcessingScripts\nzbToMedia
2016-06-18 15:37:27 DEBUG   ::MAIN: git output: 'rev-parse' is not recognized as an internal or external command,
operable program or batch file.

C:\PostProcessingScripts\nzbToMedia>
2016-06-18 15:37:27 DEBUG   ::MAIN: "C:\Program Files\Git\git-cmd.exe" rev-parse HEAD : returned successful
2016-06-18 15:37:27 ERROR   ::MAIN: Output doesn't look like a hash, not using it
2016-06-18 15:37:27 ERROR   ::MAIN: Unable to determine installed version via git, please check your logs!
2016-06-18 15:37:27 INFO    ::MAIN: No update needed
2016-06-18 15:37:27 INFO    ::MAIN: nzbToMedia Version:10.15 Branch:'symbolic-ref' is not recognized as an internal or external command,
operable program or batch file.

C:\PostProcessingScripts\nzbToMedia> (Windows 8)
2016-06-18 15:37:27 DEBUG   ::MAIN: Connecting to utorrent: http://localhost:8000/gui/
2016-06-18 15:37:27 INFO    ::MAIN: #########################################################
2016-06-18 15:37:27 INFO    ::MAIN: ## ..::[TorrentToMedia.py]::.. ##
2016-06-18 15:37:27 INFO    ::MAIN: #########################################################
2016-06-18 15:37:27 DEBUG   ::MAIN: Options passed into TorrentToMedia: ['C:\\PostProcessingScripts\\nzbToMedia\\TorrentToMedia.py', 'D:\\Downloads\\Completed\\movies\\Gravity.2013.720p.BluRay.x264-SPARKS[rarbg]', 'Gravity.2013.720p.BluRay.x264-SPARKS[rarbg]', 'movies', '76AECBD92445AB03335DD165995FACDD66D3377F']
2016-06-18 15:37:27 DEBUG   ::MAIN: Adding TORRENT download info for directory D:\Downloads\Completed\movies\Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] to database
2016-06-18 15:37:27 DB      ::MAIN: nzbtomedia.db: UPDATE downloads SET status = ?, client_agent = ?, last_update = ?, input_id = ?, input_hash = ?, input_name = ? WHERE input_directory = ? with args [0, u'utorrent', 736133, u'76AECBD92445AB03335DD165995FACDD66D3377F', u'76AECBD92445AB03335DD165995FACDD66D3377F', u'Gravity.2013.720p.BluRay.x264-SPARKS[rarbg]', u'D:\\Downloads\\Completed\\movies\\Gravity.2013.720p.BluRay.x264-SPARKS[rarbg]']
2016-06-18 15:37:27 DEBUG   ::MAIN: Received Directory: D:\Downloads\Completed\movies\Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] | Name: Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] | Category: movies
2016-06-18 15:37:27 DEBUG   ::MAIN: SEARCH: Found the Category: movies in directory structure
2016-06-18 15:37:27 INFO    ::MAIN: SEARCH: Found a unique directory Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] in the category directory
2016-06-18 15:37:27 DEBUG   ::MAIN: Determined Directory: D:\Downloads\Completed\movies\Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] | Name: Gravity.2013.720p.BluRay.x264-SPARKS[rarbg] | Category: movies
2016-06-18 15:37:27 INFO    ::MAIN: Auto-detected SECTION:CouchPotato
```

**Comment by blade316** [comment]
@clinton-hall any initial ideas on this one? .... I was originally thinking it might have been because I'm running my D: drive using Storage Spaces on ReFS, however that cant be it as the nzbToCouchPotato.py runs fine with SABnzbd which is saving to the same D: drive.

**Comment by labrys** [comment]
@blade316 This is likely related to issue #1071 and may be fixed by PR #1074 which has been merged to nightly.  Please let me know if this fixes your issue.

**Comment by blade316** [comment]
@labrys OK no worries I will update to the latest nightly when I get home, and I'll let you know
